src/mediapipe_internal/mediapipegraphdefinition.cpp

```diff
@@ -380,6 +390,15 @@ std::string MediapipeGraphDefinition::getStreamName(const std::string& streamFul
     return EMPTY_STREAM_NAME;
 }
 
+PythonNodeResource* MediapipeGraphDefinition::getPythonNodeResource(const std::string& nodeName) {
```

Comment:
This function is never used

Reply:
Use in tests. Will be used in future implementations.

Reply:
Then let's add it when we need it I think